src/test/java/com/epam/tc/hw3/pageobjects/elements/DifferentElements.java

```diff
@@ -0,0 +1,180 @@
+package com.epam.tc.hw3.pageobjects.elements;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+
+public class DifferentElements extends BaseElements {
+
+    @FindBy(xpath = "//label[@class='label-checkbox' and contains(., 'Water')]/input")
+    private WebElement checkboxWater;
+
+    @FindBy(xpath = "//label[@class='label-checkbox' and contains(., 'Earth')]/input")
+    private WebElement checkboxEarth;
+
+    @FindBy(xpath = "//label[@class='label-checkbox' and contains(., 'Wind')]/input")
+    private WebElement checkboxWind;
+
+    @FindBy(xpath = "//label[@class='label-checkbox' and contains(., 'Fire')]/input")
+    private WebElement checkboxFire;
+
```

Comment:
Could be combined to the list of web elements

Reply:
Fixed